docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-classification]]
+=== Predicting delaying flights with {classanalysis}
+
+Let's try to predict whether a flight will be delayed or not by using the 
+{kibana-ref}/add-sample-data.html[sample flight data]. We use this data to learn 
+a {classification} model that allows us to make the prediction based on 
+information such as weather conditions and location of the destination and 
+origin, flight distance and carrier.
+
+This dataset consists of sample data that has been manually created and it 
+contains inconsistencies. For example, a flight can be both delayed and 
+canceled. Nonetheless, the use case is relevant and it is a good example of how 
+the quality of input data will affect the quality of results.
+
+We create a {dfanalytics-job} to predict if a given flight is delayed. This 
```

Comment:
> We create a data frame analytics job... on the field FlightDelay.

In my opinion, these first two sentences don't fit in with this paragraph.  I'd suggest moving them after the example of the document (see subsequent comment).